products/jquery.md

```diff
@@ -0,0 +1,49 @@
+---
+title: jQuery
+layout: post
+category: framework
+sortReleasesBy: "releaseCycle"
+changelogTemplate: "https://github.com/jquery/jquery/releases/tag/__LATEST__"
+releases:
+  - releaseCycle: "3"
+    eol: false
+    release: 2016-06-09
+    latest: "3.6.0"
+  - releaseCycle: "2"
+    eol: true
+    release: 2013-04-18
+    latest: "2.2.4"
+  - releaseCycle: "1"
+    eol: true
+    release: 2006-08-31
+    latest: "1.12.4"
+iconSlug: jquery
+permalink: /jquery
+activeSupportColumn: false
+releaseColumn: true
+releaseDateColumn: true
+---
+> [jQuery](https://jquery.com/) is a widely used JavaScript library making it easier to manipulate HTML documents.
+
+The 1.x and 2.x branches no longer receive patches.
+
+## [Browser Support](https://jquery.com/browser-support/)
+
+### Desktop
+
+| Browser               | Support Versions               |
+|-----------------------|--------------------------------|
+| Chrome                | (Current - 1) and Current      |
+| Edge                  | (Current - 1) and Current      |
+| Firefox               | (Current - 1) and Current, ESR |
+| Chrome                | (Current - 1) and Current      |
+| Internet Explorer     | 9+                             |
+| Safari                | (Current - 1) and Current      |
+| Opera                 | Current                        |
```

Comment:
Not entirely sure about the wording, but `(Current, Current-1)` is very vague. Also removed Chrome, which was repeated in the table.

```suggestion
| Chrome                | Latest 2 major stable releases |
| Edge                  | Latest 2 major stable releases |
| Firefox               | Latest 2 major stable releases, [ESR](https://support.mozilla.org/kb/firefox-esr-release-cycle) |
| Internet Explorer     | 9+                             |
| Safari                | Latest 2 major stable releases      |
| Opera                 | Latest stable version                        |
```

Reply:
I used the same wording as the jQuery website but I agree it is confusing.